mtags/src/main/scala-2/scala/meta/internal/pc/completions/MatchCaseCompletions.scala

```diff
@@ -38,7 +38,7 @@ trait MatchCaseCompletions { this: MetalsGlobal =>
       parent: Tree,
       patternOnly: Option[String] = None,
       hasBind: Boolean = false,
-      includeExhaustive: Boolean = false
+      includeExhaustive: Option[(Boolean, Boolean)] = None
```

Comment:
```suggestion
      includeExhaustive: Option[NewLineOptions] = None
```
maybe and additional case class to avoid tuple of booleans?